test/arrays/ferguson/from-iterator/skyline.chpl

```diff
@@ -0,0 +1,21 @@
+config const n = 2;
```

Comment:
I suggest adding the phrases "skyline array" and "jagged array" into the body of this test. That way this test will will be picked up by grep.